back/lib/participation_method/base.rb

```diff
@@ -125,6 +125,10 @@ def supports_survey_form?
       false
     end
 
+    def supports_everyone_permission?
+      false
+    end
+
```

Comment:
If I'm being picky, maybe this method should be called `supports_permitted_by_everyone` given the field name is called `permitted_by`

Reply:
I agree `supports_permitted_by_everyone?` is a better name.